atlp-rwanda/rca-barfoot-team3-bn
# #184637933 Ch support multiple languages

#### What does this PR do?
Adds languages accessibility 

#### Description of Task to be completed?
Allowing the user to change the language across the app

#### How should this be manually tested?
Once the project is cloned, run it and open postman

- In the "headers tab", add a new header with the key "Accept-Language" and the value of the language you want to test (eg: fr for French or en for English)
- Then send the request
- Inspect the response in the "Body" tab. You should see a JSON object containing the welcome message and the language that was detected by the server.
- Repeat the test with a different language by changing the "Accept-Language" header value.

#### What are the relevant pivotal tracker/Trello stories?
#184637933
#### Screenshots (if appropriate)
-For French
![image](https://user-images.githubusercontent.com/73661281/227799688-9b1bb6ad-7477-4055-8cbc-8da28a0bfa5a.png)

-For English
![image](https://user-images.githubusercontent.com/73661281/227799764-1186cf1b-2f4d-4fe1-b332-64de3dfb8fd3.png)
